templates/css/styles.css

```diff
@@ -211,19 +206,135 @@ header h3 {
   background-position: right 8px center;
   background-repeat: no-repeat;
   background-image: -webkit-image-set(
-    url(/img/ic_play_circle_filled_black_1x_web_24dp.png) 1x,
-    url(/img/ic_play_circle_filled_black_2x_web_24dp.png) 2x
+  url(/img/ic_play_circle_filled_black_1x_web_24dp.png) 1x,
+  url(/img/ic_play_circle_filled_black_2x_web_24dp.png) 2x
   );
 }
 .exit {
   background-position: right center;
   background-repeat: no-repeat;
   background-image: -webkit-image-set(
-    url(/img/ic_close_black_1x_web_24dp.png) 1x,
-    url(/img/ic_close_black_2x_web_24dp.png) 2x
+  url(/img/ic_close_black_1x_web_24dp.png) 1x,
+  url(/img/ic_close_black_2x_web_24dp.png) 2x
   );
 }
 
+/* nav bar */
+nav {
+  background-color: #4E4E4E;
+  box-shadow: 0 2px 2px 0 rgba(0,0,0,.14),0 3px 1px -2px rgba(0,0,0,.2),0 1px 5px 0 rgba(0,0,0,.12);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+}
+nav #logo {
+  align-self: flex-start;
+  margin: 0 auto;
+  text-align: left;
+  font-weight: 200;
+  font-size: 38px;
+  text-transform: uppercase;
+  line-height: 36px;
+  color: white;  
+  padding-left: 44px;
+  background-position: left center;
+  background-repeat: no-repeat;
+  background-image: -webkit-image-set( url(/img/logo-icon_1x.png) 1x, url(/img/logo-icon_2x.png) 2x );
+}
+nav #hamburger {
+  align-self: flex-start;
+  padding-top: 8px;
+  padding-left: 8px;
+}
+nav #actions {
+  max-width: 50%;
+  padding: 0 16px;
+  align-self: flex-start;
+  margin-left: auto;
+}
+nav .button {
+  padding: 0px;
+  height: 40px;
+  min-width: 40px;
+  display: block;
+  float: left;
+}
+nav .button > amp-img {
+  margin: 8px;
+  float: left;
+}
+nav .button > span {
+  display: inline-block;
+  font-weight: 300;
+  line-height: 40px;
+  color: white;  
+  text-transform: uppercase;
+}
+
+/* sidebar menu */
+#sidebar {
+  width: 320px;
+}
+.sidebar-close-image {
+   float:right;
+   top: 24px;
+   margin-right: 16px;
+}
+.sidebar-close-image:hover {
+    background-color: #ccc;
+}
+.list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  margin-left: 32px;
+}
+.listitem {
+  font-weight: 400;
+  list-style: none;
+  font-size: 14px;
+  height: 48px;
+  padding: 0;
+  color: rgba(0,0,0,.87);
+}
+.selected {
+  font-weight: 700;
+  color: #ff4081;
+}
+.categoryheader {
```

Comment:
use a dash to separate different words => category-header. Also applies to topheader, homelogo, listgrouping,..